### DISCUSSION TITLE
Timeout Problem

### DISCUSSION THREAD

**Discussion by lhx077** [community/community#14327] [Copilot Conversations]
I've joined the github copilot waitlist and qualified for the closed beta, but I can't use it, it prompts me: GitHub Copilot could not connect to server. Extension activation failed: "connect ETIMEDOUT 54.169.195.247:443，how can I solve this problem？？？

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?
